slash/css/src/Message/Message.css

```
@@ -0,0 +1,80 @@
+.af-alert {
+  --alert-sidebar-color: var(--red30);
+  --alert-background-color: var(--red10);
+  --alert-border-color: var(--red20);
+
+  display: flex;
+  border: 1px solid var(--alert-border-color);
+  flex-direction: row;
+  gap: 20px;
```

histoire des cohérent avec la parti client il faudrait utiliser la fonction calc non ?
```suggestion
  gap: calc(20rem / 16);
```

---

corrigé